packages/cashier/src/stores/__tests__/crypto-fiat-converter-store.spec.ts

```diff
@@ -1,9 +1,8 @@
-import { TRootStore, TWebSocket } from 'Types';
+import { TWebSocket } from '../../types';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻